src/components/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class AddTodo extends React.Component {
+  state = {
+    title: '',
+    id: 0,
+  }
+
+  handleInputTodo = ({ target }) => {
+    this.setState({
+      title: target.value.trimStart(),
```

Comment:
юзер может случайно нажать пробел и в конце строки
```suggestion
      title: target.value.trim(),
```